Formula/ntfs-3g.rb

```diff
@@ -38,25 +35,23 @@ class Ntfs3g < Formula
   end
 
   on_linux do
-    depends_on "libfuse"
+    depends_on "libfuse@2"
   end
 
   def install
-    ENV.append "LDFLAGS", "-lintl"
+    on_macos { ENV.append "LDFLAGS", "-lintl" }
```

Comment:
```suggestion
```

The formula is disabled for macOS, don't think we need this line at all

Reply:
I was wondering if we wanted to keep some macOS-steps for anyone who maintains their own tap and syncs updates from core.

Alternatively, we can just remove all macOS-specific steps and even consider marking fuse-based formulae as `depends_on :linux` (it may help to auto-label PRs).

On similar note, the `mount_ntfs` script that is created during install may not work correctly on Linux, but I am not familiar with it, so I decided to avoid modifying it.

Reply:
Actually, it may be installed on macOS with `--force`